lab_5_scrapper/scrapper.py

```diff
@@ -187,6 +314,21 @@ def unify_date_format(self, date_str: str) -> datetime.datetime:
         Returns:
             datetime.datetime: Datetime object
         """
+        months = {"января": "Jan", "февраля": "Feb", "марта": "Mar",
+                  "апреля": "Apr", "мая": "May", "июня": "Jun", "июля": "Jul",
+                  "августа": "Aug", "сентября": "Sep", "октября": "Oct", "ноября": "Nov",
+                  "декабря": "Dec"}
+        if ":" in date_str:
+            today = datetime.date.today()
+            hour, minute = map(int, date_str.split(":"))
+            d = datetime.datetime(year=today.year, month=today.month,
+                                  day=today.day, hour=hour,
+                                  minute=minute)
+        else:
+            day, mon, year = date_str.replace(',', '').split(' ')
+            d = f'{day} {months[mon]} {year}'
+            d = datetime.datetime.strptime(d, "%d %b %Y")
+        return d
```

Comment:
some mypy errors here:

lab_5_scrapper/scrapper.py:329: error: Incompatible types in assignment (expression has type "str", variable has type "datetime")  [assignment]
lab_5_scrapper/scrapper.py:330: error: Argument 1 to "strptime" of "datetime" has incompatible type "datetime"; expected "str"  [arg-type]

you try to assign different types (str/datetime) to your d variable 

see
 line 324 -> d type is datetime
 line 329 -> d type is str
 line 330 -> d type is datetime

you may use different variables for these two assignments

Reply:
thank you!!